### PR TITLE
Add example for usage with logstash

### PR DESCRIPTION
**Pull request by tarjei**
It irritated me that I couldn't find one concrete example in the docs - at least not where I was looking.

### CODE REVIEW

**Comment by untergeek** [comment]
I'm sorry you couldn't find an example that suited you.

If the examples in the guide do not satisfy you, this would be the correct place to add one, rather than as an FAQ:

http://www.elastic.co/guide/en/elasticsearch/client/curator/current/examples.html

Also, please see http://www.elastic.co/guide/en/elasticsearch/client/curator/current/_how_to_contribute.html for the contribution guide.

**Comment by untergeek** [comment]
There's also the [site corrections](http://www.elastic.co/guide/en/elasticsearch/client/curator/current/site-corrections.html) page.

File: docs/asciidoc/misc/faq.asciidoc [hunk 1]
```diff
@@ -14,6 +14,15 @@ A: Use the "Edit" link on any page
 
 See <<site-corrections,Site Corrections>>.
 
+== Q: How can I delete logstash indexes older then 35 days?
+
+A: Like this:
+~~~~~~~~~~~~~~~~~~~
+
+```
+$ curator delete indices --older-than 35 --time-unit  days --timestring '%Y.%m.%d'
```
Comment (untergeek) [inline docs/asciidoc/misc/faq.asciidoc]: Technically, this is not Logstash specific until you add `--prefix logstash`

**Comment by untergeek** [comment]
And the Examples page is a sub-page under [Getting Started](http://www.elastic.co/guide/en/elasticsearch/client/curator/current/getting-started.html), if you were wondering.

**Comment by tarjei** [comment]
Hi, I have just signed the contributors agreement, but I would appreciate
if you could just move the example yourself as I will probably spend quite
some time reformatting the command etc :)

Thanks for a nice tool.

2015-04-12 19:18 GMT+02:00 Aaron Mildenstein <notifications@github.com>:

> And the Examples page is a sub-page under Getting Started
> <http://www.elastic.co/guide/en/elasticsearch/client/curator/current/getting-started.html>,
> if you were wondering.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/curator/pull/344#issuecomment-92089205>.
>



-- 
__________________
Tarjei Huse
Mobil: 920 63 413

**Comment by untergeek** [comment]
That would be a bit redundant, since there is little difference between what you've added and the first example on the [Examples](http://www.elastic.co/guide/en/elasticsearch/client/curator/current/examples.html) page, which reads:

> ## Delete _all_ indices with matching --timestring which are older than 30 days
>
>     curator --host 10.0.0.2 delete indices --older-than 30 --time-unit days --timestring '%Y.%m.%d'`
> 
>
>In previous versions of Curator, you would have needed as many runs of the command-line as you had prefixes, no matter the matching pattern. Not so any more! With this example, all indices with a pattern of `%Y.%m.%d` inside will be deleted. This is useful for managing both Logstash and Marvel indices with a single command, if you want the same retention period.

The only differences are that the existing example has a `--host` flag, and it's 30 days vs. 35.

I'd be amenable to another example, adding `logstash` as a `--prefix`.

**Comment by tarjei** [comment]
Ah, ok, odd I didn't find that :(

Thanks for helping!

2015-04-12 19:41 GMT+02:00 Aaron Mildenstein <notifications@github.com>:

> That would be a bit redundant, since there is little difference between
> what you've added and the first example on the Examples
> <http://www.elastic.co/guide/en/elasticsearch/client/curator/current/examples.html>
> page, which reads:
>
> Delete *all* indices with matching --timestring which are older than 30
> days
>
> curator --host 10.0.0.2 delete indices --older-than 30 --time-unit days --timestring '%Y.%m.%d'`
>
> In previous versions of Curator, you would have needed as many runs of the
> command-line as you had prefixes, no matter the matching pattern. Not so
> any more! With this example, all indices with a pattern of %Y.%m.%d
> inside will be deleted. This is useful for managing both Logstash and
> Marvel indices with a single command, if you want the same retention period.
>
> The only differences are that the existing example has a --host flag, and
> it's 30 days vs. 35.
>
> I'd be amenable to another example, adding logstash as a --prefix.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/curator/pull/344#issuecomment-92093721>.
>



-- 
__________________
Tarjei Huse
Mobil: 920 63 413

**Comment by untergeek** [comment]
I will close this issue, then.  Feel free to re-open, or open a new one if you like.

**Pull request closed without merging**